x/examples/outline-pwa/README.md

```diff
@@ -1,19 +1,346 @@
-# Outline PWA Wrapper
+# Building a Censorship-Resistant Android/iOS App with CapacitorJS and the Outline SDK Mobileproxy
 
-Demonstration of how to wrap any URL with CapacitorJS in a way that all content is loaded over Outline SDK's Mobileproxy. Turn your website into a censorship-resistant app!
+This code lab guides you through creating a censorship-resistant Android/iOS app from scratch that wraps your Progressive Web App (PWA) using CapacitorJS and the Outline SDK Mobileproxy. This setup routes your site's traffic through the Outline proxy, bypassing network restrictions. Note that the source in this folder is representative of what your end product should look like once the code lab is completed.
 
-TODO: step-by-step walkthrough of the app-building process
+**Prerequisites:**
 
-## Setup
+* An existing PWA
+* Make sure your development environment is set up with the following. [You can also follow CapacitorJS's environment setup guide](https://capacitorjs.com/docs/getting-started/environment-setup)
+  * [Node.js](https://nodejs.org/en/)
+  * [GoLang](https://go.dev/)
+  * For Android, [OpenJDK 17](https://stackoverflow.com/a/70649641) and [Android Studio](https://developer.android.com/studio/)
+  * For iOS, [XCode](https://developer.apple.com/xcode/) and [cocoapods](https://cocoapods.org/)
+* [Wireshark](https://www.wireshark.org/), to confirm the app is working
+
+> [!TIP]
+> Kotlin Multiplatform [provides a tool called `kdoctor`](https://github.com/Kotlin/kdoctor) you can use to verify that your system is set up for cross-platform mobile development. You can ensure that
+> your system has the required dependencies by [installing it](https://github.com/Kotlin/kdoctor?tab=readme-ov-file#installation) and running `kdoctor`.
+
+## Set up the Capacitor Project
+
+* Follow the CapacitorJS Getting Started guide to initialize a new project: [https://capacitorjs.com/docs/getting-started](https://capacitorjs.com/docs/getting-started)
+
+> [!NOTE]
+> This will create a new directory containing your project. Make sure you run these commands relative to where you want your project to live.
+
+   ```bash
+   npm init @capacitor/app # follow the instructions - make sure to pick an app ID that's unique!
+   cd <my-app-dir>
+   npm install
+   ```
+
+* Add iOS and Android platforms to your project:
+
+  ```bash
+  npm install @capacitor/android @capacitor/ios
+  npx cap add android
+  npx cap add ios
+
+  mkdir android/app/src/main/assets # sync will fail without this folder
+  npx cap sync
```

Comment:
I'm getting npx cap sync android
[error] Could not find the web assets directory: ./www.

After I create:
 npx cap sync android
[error] The web assets directory (./www) must contain an index.html file.
        It will be the entry point for the web portion of the Capacitor app.